python-avd/pyavd/_eos_designs/structured_config/underlay/ethernet_interfaces.py

```diff
@@ -299,6 +299,7 @@ def ethernet_interfaces(self: AvdStructuredConfigUnderlay) -> list | None:
                     "description": "DIRECT LAN HA LINK",
                     "ip_address": self.shared_utils.wan_ha_ip_addresses[index],
                     "flow_tracker": direct_wan_ha_links_flow_tracker,
+                    "mtu": self.shared_utils.configured_wan_ha_mtu,
```

Comment:
Note that this concerns only WAN HA direct link and not the case where uplinks are used as WAN HA interfaces -> in that case AVD would still configure 9214 by default with this PR right?
@ClausHolbechArista should we adjust the MTU dynamically if the interface is being used as WAN HA? (this would be a bit more complex to implement as there is a way to exclude some uplink interfaces from WAN HA)

Reply:
So we need to update the platform settings for the WAN platforms as well to do this.

Reply:
updated